testing/trino-benchto-benchmarks/src/test/resources/sql/presto/tpcds/q91.plan.txt

```diff
@@ -6,27 +6,27 @@ remote exchange (GATHER, SINGLE, [])
+                                                                                join (INNER, REPLICATED):
+                                                                                    scan customer
+                                                                                    local exchange (GATHER, SINGLE, [])
+                                                                                        remote exchange (REPLICATE, BROADCAST, [])
+                                                                                            scan customer_address
                                 local exchange (GATHER, SINGLE, [])
                                     remote exchange (REPLICATE, BROADCAST, [])
-                                        join (INNER, REPLICATED):
-                                            scan household_demographics
-                                            local exchange (GATHER, SINGLE, [])
-                                                remote exchange (REPLICATE, BROADCAST, [])
-                                                    join (INNER, REPLICATED):
-                                                        scan customer_demographics
-                                                        local exchange (GATHER, SINGLE, [])
-                                                            remote exchange (REPLICATE, BROADCAST, [])
-                                                                join (INNER, REPLICATED):
-                                                                    scan date_dim
-                                                                    local exchange (GATHER, SINGLE, [])
-                                                                        remote exchange (REPLICATE, BROADCAST, [])
-                                                                            join (INNER, REPLICATED):
-                                                                                scan catalog_returns
-                                                                                local exchange (GATHER, SINGLE, [])
-                                                                                    remote exchange (REPLICATE, BROADCAST, [])
-                                                                                        join (INNER, REPLICATED):
-                                                                                            scan customer
-                                                                                            local exchange (GATHER, SINGLE, [])
-                                                                                                remote exchange (REPLICATE, BROADCAST, [])
-                                                                                                    scan customer_address
+                                        scan household_demographics
```

Comment:
cc @sopel39

Reply:
Why did it rotate? Could you benchmark this query?

Reply:
I benchmarked q91 on unpartitioned and partitioned ORC data.
There was small reduction in wall time, and no change in CPU time.